interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,174 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
+
+On Linux:
+`export VARIABLE=value`
+These can be set automatically in VSCode in an .env file, run with `source .env` in Linux or made into a `.bat` file to easily run from the command line in Windows. You can ask a team member for a .env file configured for our development environment to save time.
+
+Create a .azureml/config.json file in the project's root directory that looks like so:
+```json
+{
+"subscription_id": "<subscription id>",
+"resource_group": "<resource group>",
+"workspace_name": "<workspace name>"
+}
+
+```
+
+## Training Pipeline
+Here's an example of a possible config file:
```

Comment:
Could you please include a bit more information about what the option are and which ones are likely to be modified by the user? I can eyeball the file and guess what it's doing, but for a geophysicist this might be hard to do

Reply:
You could also point to AML documentation if it's easier

Reply:
Right now we are only set up to run the MpiStep (train). I agree we should add a link here to the AML documentation, clearing up the steps that we are contributing in a separate conversation so we can reduce confusion here.

Reply:
I also realized that this is a different config file - it's actually pipeline_config.json? Then just say "Here's an example of a possible pipeline_config.json file" and explain how it's used and where it's placed

Reply:
Updated config to pipeline configuration file in commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/985d54b411d4fed07055fec9106ea56a2a5924c6
Pointing to docs on AzureML pipeline steps in this commit: https://github.com/microsoft/seismic-deeplearning/commit/8e5cc721ee55b8deb55ddd9d4c11a238853e82c5 Modifying the example in the README.md to only include the training step that we use in this commit: https://github.com/microsoft/seismic-deeplearning/commit/da15de2b6b7eacb04dc79e80ca1f66811d74495e